public/worker.js

```diff
@@ -0,0 +1,45 @@
+const CACHE_NAME = "react play";
```

Comment:
I would put the cache name without a space.

Reply:
Okay, I will fix that